mxfusion/components/functions/operators/operators.py

```diff
@@ -0,0 +1,72 @@
+from ..function_evaluation import FunctionEvaluation, FunctionEvaluationDecorator
+from ...variables import Variable
+
+class Operator(FunctionEvaluation):
+    """
+    Abstract Operator object for using MXNet operators in MXFusion space.
+    Child classes implement the eval method with their operator and access necessary state through the properties dictionary.
+    """
+    def __init__(self, inputs, outputs, properties=None):
```

Comment:
Add another argument for the operator name, which will be the identified of the operator. It will be helpful for debug and serialization.